tests/e2e/gc_file_share_negative.go

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
There is a test similar to this I believe which runs in the block vanilla BR pipeline - verify dynamic volume provisioning fails for VSAN datastore in datacenter doesn't have VSAN FS enabled

can you please see is that test fits the needs or needs some enhancements